client/landing/subscriptions/components/tabs-switcher/tabs-switcher.tsx

```diff
@@ -48,13 +46,7 @@ const TabsSwitcher = () => {
 					</NavItem>
 
 					<NavItem
-						onClick={ () => {
-							shouldEnableCommentsTab
-								? navigate( commentsPath )
-								: window.location.replace(
-										`https://wordpress.com/email-subscriptions/?option=comments&locale=${ locale }`
-								  );
-						} }
+						onClick={ () => navigate( commentsPath ) }
```

Comment:
This is enabling the "Comments" page for logged-in users. We were meant to release this tomorrow, but I guess it is fine. Just, there will be no heads-up post related to this.

Reply:
BTW I just realized the Comments page is only released to English external users so far. Therefore, we need to redirect all locale external users to `/subscriptions/comments`, when they access `/email-subscriptions/?options=comments`. This needs to be done in WPCOM.